build/test.py

```diff
@@ -0,0 +1,22 @@
+import covidsim as cs
+
+f = cs.Factory()
+population = f.make_population()
+
+f.add_cells(population, 5)
```

Comment:
Agree, having it in the factory is a little dodgy.

Came down to wanting to make it hard to add or remove cells, microcells and people mid simulation running to avoid having to do extra checks about the size of the vectors at each iteration.

I think these methods are going to be made private to be used as population helper functions, and this PopulationFactory class is instead going to just expose make_population(n_cells, n_microcellsPerCell, n_peoplePerMicrocell) which will construct a basic population and return it.

Maybe in future we can also create a PopulationFactoryFromData class which will read in a datafile and construct the population from that.